src/Logger.ts

```diff
@@ -0,0 +1,31 @@
+const ESC = "\u001b"
+const RESET = ESC + "[m"
+const GRAY  = ESC + "[37m"
+const WHITE = ESC + "[97m"
+const DARK  = ESC + "[90m"
+const RED   = ESC + "[91m"
+
+export default class Logger {
+
+	constructor(private readonly actor: string) {}
+	
+	log(message: string) {
+		console.log(`${WHITE}[${DARK}${time()}${WHITE}] [${GRAY}${this.actor}${WHITE}]${GRAY}`, message)
+	}
+
+	error(message: string) {
+		console.log(`${WHITE}[${DARK}${time()}${WHITE}] [${RED}${this.actor}${WHITE}]${RED}`, message)
+	}
+
```

Comment:
What about a warn() and debug() function like you told me on Discord?

Reply:
addressed by: [df3c529](https://github.com/creelonestudios/mailverse/pull/9/commits/df3c529f72307e04790a33966149ff4d42d04a79)